portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/passwordpolicies/PasswordPolicies.testcase

```diff
@@ -541,6 +541,271 @@ definition {
 			userEmailAddress = "userea@liferay.com");
 	}
 
+	@description = "This is a use case for LPS-193886 (TC-2). A new User cannot set their new password through the password setup link, if it is expired."
+	@priority = 5
+	test NewUserCannotSetNewPasswordThroughTheExpiredLink {
+		property custom.properties = "feature.flag.LPS-193884=true${line.separator}passwords.default.policy.change.required=false${line.separator}passwords.default.policy.reset.ticket.max.age=300";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given: A new User is created and Admin User logs out") {
+			var portalURL = PropsUtil.get("portal.url");
+
+			PasswordPolicies.openPasswordPoliciesAdmin();
+
+			PasswordPolicies.enableChangeable(
+				enableChangeRequired = "false",
+				passwordPolicyName = "Default Password Policy",
+				passwordPolicyPanel = "Password Changes",
+				resetTicketMaxAge = "5 Minutes");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Users",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			User.addCP(
+				userEmailAddress = "b@liferay.com",
+				userFirstName = "b",
+				userLastName = "b",
+				userScreenName = "b");
+
+			User.logoutPG();
+		}
+
+		task ("When: The link in the received email is opened after it expired (wait for it to expire)") {
+			Pause(value1 = 310000);
```

Comment:
We need to add _pause_ very carefully. 
Please try to see if you can reduce the pause time by shortening the expiration time(aka passwords.default.policy.reset.ticket.max.age=300). 
If not, I'm okay with the long pause.
In addition, when adding a pause, we need to add a description to explain the necessity of this pause. Please refer to:
https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase#L976-L978

Reply:
Sadly, there's no way to work around this one as we are just waiting for the setup link to expire and it has no visible feeedback. I'll add the explanation to the test cases.